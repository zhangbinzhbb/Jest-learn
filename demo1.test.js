import { generateConfig } from './demo1'

test('测试 generateConfig 函数',()=>{
    expect(generateConfig()).toMatchSnapshot();
})