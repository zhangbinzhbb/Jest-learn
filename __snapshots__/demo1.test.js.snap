// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 generateConfig 函数 1`] = `
Object {
  "port": 8080,
  "server": "http://localhost",
  "time": 2020-07-15T08:17:41.803Z,
}
`;
